test/spec/modules/firstPartyData_spec.js

```diff
@@ -0,0 +1,594 @@
+import {expect} from 'chai';
+import * as utils from 'src/utils.js';
+import {config} from 'src/config.js';
+import { getGlobal } from 'src/prebidGlobal.js';
+import CONSTANTS from 'src/constants.json';
+import {getRefererInfo} from 'src/refererDetection.js';
+import {
+  filterData,
+  validateFpd,
+  init,
+  resetOrtb2
+} from 'modules/firstPartyData/index.js';
+import events from 'src/events.js';
+
+describe('the first party data module', function () {
+  let sandbox,
+    logErrorSpy;
```

Comment:
`logErrorSpy` variable is not used anywhere in this file...better remove it?

Reply:
Removing